### ISSUE TITLE
Code deprecation issues when running tests.

### ISSUE THREAD

**Issue by stephen-cox** [localgovdrupal/localgov_services#23]
When running the tests there are lots of deprecated code issues. Most are related to contrib modules, but there's an issues with some of the tests provided by the localgov_drupal module.
```  
2x: Drupal\Tests\BrowserTestBase::$defaultTheme is required in drupal:9.0.0 when using an install profile that does not set a default theme. See https://www.drupal.org/node/3083055, which includes recommendations on which theme to use.
    1x in PagesIntegrationTest::testConfigForm from Drupal\Tests\localgov_services\Functional
    1x in PagesIntegrationTest::testPostLink from Drupal\Tests\localgov_services\Functional
```

**Comment by ekes** [comment]
Assume this got fixed by 6ad678c4

**Comment by ekes** [comment]
And another different one related to config exported into services 7cedab5
